src/app/overview/widgets/stakinginfo/stakinginfo.component.ts

```diff
@@ -67,4 +67,21 @@ export class StakinginfoComponent implements OnInit {
     this.curWeight = Math.floor(this.curWeight);
     this.log.d(`calculateDynamicStakingReward, dynamicStakingReward = ${this.dynamicStakingReward}`);
   }
+
+  getBeforeDot(int: number): string {
+    if ((int + '').indexOf('.') >= 0) {
```

Comment:
Make use of toString and use a variable, as well as a single inline return. This is way ineffecient for no reason